examples/tabular/tabular-fasttext.md

```diff
@@ -0,0 +1,147 @@
+```python
```

Comment:
Could you change this to be a python notebook? An example would be the SHAP tutorials in `examples/tabular/interpret/`. I think we want to change all our `.md` tutorials to notebook tutorials eventually.